build.Dockerfile

```diff
@@ -55,14 +58,20 @@ ENV ADDITIONAL_FEATURES=$ADDITIONAL_FEATURES_ARG
 
 ARG FINGERPRINT=none
 
+ARG SGX_COMMERCIAL_KEY=enclave-runtime/Enclave_private.pem
+ENV SGX_COMMERCIAL_KEY ${SGX_COMMERCIAL_KEY}
```

Comment:
As you wish;)

Reply:
Just for future reference:

We had a discussion about this, and we decided that this will be left as it is, because getting rid of that `SGX_COMMERCIAL_KEY` might be a bit more involved.